app/controllers/vinyls_controller.rb

```diff
@@ -4,20 +4,23 @@ def index
   end
 
   def show
-    @vinyl = Vinyl.find(params[:id])
+    @user = User.find(params[:user_id])
+    @vinyl = @user.vinyls
     @reviews = @vinyl.reviews
-    @review = @vinyl.reviews.build
+    @review = @vinly.reviews.build
```

Comment:
thats why this probably isn't working because u are returning an array of vinyls and then doing reviews.buld on an array :+1: